Quest Behaviors/ForcedMount.cs

```diff
@@ -110,12 +110,19 @@ public ForcedMount(Dictionary<string, string> args)
 
             else
             {
-                Mount.FlyingMounts.First().CreatureSpell.Cast();
+                if (!string.IsNullOrEmpty(Styx.Helpers.CharacterSettings.Instance.FlyingMountName))
+                {
+                    foreach (var mount in Mount.FlyingMounts.Where(mount => mount.Name == Styx.Helpers.CharacterSettings.Instance.FlyingMountName))
+                        Mount.FlyingMounts.FirstOrDefault(m => m.CreatureSpellId == mount.CreatureSpellId)?.CreatureSpell.Cast();
+                }
+                else
+                    Mount.FlyingMounts.First().CreatureSpell.Cast();
                 await Coroutine.Wait(3000, () => StyxWoW.Me.Mounted);
             }
 
             // Hop off the ground. Kthx
-            await Coroutine.Sleep(2500);
+            await Coroutine.Sleep(250);
+            await Coroutine.Wait(2000, () => StyxWoW.Me.Mounted);

```

Comment:
There is already a wait for this on line 120. Removing this line and keeping the 250 ms sleep would be better.